app/views/widgets/onlineTestProgress.scala.html

```diff
@@ -36,10 +36,12 @@
             @invigilatedETraySection()
         } else {
             <p id="etrayDescription" class="">This section is timed, after starting the section you'll have 80 minutes to complete it. If we've agreed you require extra time, this will be added to the 80 minutes.</p>
-            <p id="etrayTime" class="">You have <strong>@{
-                phase2Test.getDuration
-            }</strong> to finish
-                the e-tray</p>
+            <p id="etrayTime" class="no-btm-margin">
+                You have <strong>@{ phase2Test.getDuration }</strong> to finish the e-tray.
+            </p>
+            <p id="etrayExpires">
+                It expires at <strong id="etrayExpiryTime">@{ phase2Test.getExpireTime }</strong> (London) on <strong id="etrayExpiryDate">@{phase2Test.getExpireDate}</strong>.
```

Comment:
@WarFox you need to convert this time to London timezone no?

Reply:
And date, i imagine!

Reply:
Date is not affected by this, but you are right about time. I have made that change and added a test for it